src/Model/PhasorDynamics/SynchronousMachine/GENSALwS/README.md

```diff
@@ -12,66 +19,80 @@
-- $X_{q}$ - Quadrature axis synchronous reactance, (0.5)
-- $X'_{q}$ - Quadrature axis transient reactance, (0.47619)
-- $X''_{q}$ - Quadrature axis sub-transient reactance, (0.18)
-- $T'_{d0}$ - Open circuit direct axis transient time const., (7)
-- $T''_{d0}$ - Open circuit direct axis sub-transient time const., (0.04)
-- $T'_{q0}$ - Open circuit quadrature axis transient time const., (0.75)
-- $T''_{q0}$ - Open circuit quadrature axis sub-transient time const., (0.05) 
-- $S_{10}$ - Saturation factor at 1.0 pu flux, (0) 
-- $S_{12}$ - Saturation factor at 1.2 pu flux, (0) 
-
-### Auxillary Parameters
-Transformed parameters used during implementation and for readability.
+Symbol      | Units   | Description                     | Typical Value | Note
+------------|---------|---------------------------------|---------------| ------
+$\omega_0$  | [rad/s] | synchronous frequency           | $2\pi60$
+$H$         | [s]     | rotor inertia                   | 3
+$D$         | [p.u.]  | damping coefficient             | 0
+$R_a$       | [p.u.]  | winding resistance              | 0
+$X_{\ell}$   | [p.u.] | Stator leakage reactance | 0.15 | 
+$X_{d}$   | [p.u.] | Direct axis synchronous reactance | 2.1 | 
+$X'_{d}$   | [p.u.] | Direct axis transient reactance | 0.2 | 
+$X''_{d}$   | [p.u.] | Direct axis sub-transient reactance | 0.18 | 
+$X_{q}$   | [p.u.] | Quadrature axis synchronous reactance | 0.5 | 
+$T'_{d0}$   | [s] | Open circuit direct axis transient time const. | 7 | 
+$T''_{d0}$   | [s] | Open circuit direct axis sub-transient time const. | 0.04 | 
+$T''_{q0}$   | [s] | Open circuit quadrature axis sub-transient time const. | 0.05 | 
+$S_{10}$   | [p.u.] | Saturation factor at 1.0 pu flux | 0 | 
+$S_{12}$   | [p.u.] | Saturation factor at 1.2 pu flux | 0 | 
+
+### Model Derived Parameters
 ``` math
 \begin{aligned}
   G   &=\dfrac{R_a}{R_a^2+(X''_q)^2}&
   B   &= -\dfrac{X''_q}{R_a^2+(X''_q)^2}\\
   S_A &= \dfrac{1.2\sqrt{S_{10}/S_{12}} +1}{\sqrt{S_{10}/S_{12}} +1} & 
   S_B &= \dfrac{1.2\sqrt{S_{10}/S_{12}} -1}{\sqrt{S_{10}/S_{12}} -1} \\
-  X_{d1} &= X_d-X_d'      & X_{q1} &= X_q-X_q' \\
-  X_{d2} &= X_d'-X_\ell  & X_{q2} &= X_q'-X_\ell\\
-  X_{d3} &= (X_d'-X_d'')/X_{d2}^2 & X_{q3} &= (X_q'-X_q'')/X_{q2}^2 \\
-  X_{d5} &= (X_d''-X_\ell)/X_{d2}    & X_{q5} &= (X_q''-X_\ell)/X_{q2}\\
+  X_{d1} &= X_d-X_d'       \\
+  X_{d2} &= X_d'-X_\ell & X_{q2} = X_q-X''_q \\
+  X_{d3} &= (X_d'-X_d'')/X_{d2}^2  \\
+  X_{d5} &= (X_d''-X_\ell)/X_{d2}  \\
   X_{qd} &= (X_q-X_\ell)/(X_d-X_\ell)
 \end{aligned}
 ```
 
-## Equations
+## Model Variables
+
+### Internal Variables
+
+#### Differential
+
+Symbol      | Units   | Description                     | Note
+------------|---------|---------------------------------| ------
+$\delta$  | [rad] | Machine internal rotor angle |
+$\omega$  | [p.u.] | Machine relative speed | Optionally read by governor or stabilizer component
+$\psi'_d$ | [p.u.] | Direct axis subtransient flux | 
```

Comment:
For better clarity, I would suggest defining $\omega$ as the machine speed in [p.u.] rather than relative speed. The relative speed would typically be $\Delta\omega$, but that notation might be cumbersome. 
```suggestion
$\omega$  | [p.u.] | Machine speed | Optionally read by governor or stabilizer component
$\psi'_d$ | [p.u.] | Direct axis subtransient flux | 
```